### PR TITLE
upgrade to returns 14

### LINKED ISSUE #34: Upgrade to Returns 0.14
Returns library is what we use to get functional programmings Modals into python. They just did a nice update but some stuff like @pipeline is deprecated.

So upgrade to returns 0.14

### PR DESCRIPTION
**Pull request by thomasborgen**
closes: #34 

semver will be major since this is breaking.

### CODE REVIEW

**Comment by thomasborgen** [comment]
I think we should start a changelog.md now and add it before we merge.
And this merge will mark version 1.0.0, but only because it breaks backwards compatibility and not that its a "milestone"

File: storage_bucket/get.py [hunk 1]
```diff
@@ -1,46 +1,43 @@
...
+            )),
         )
 
     @safe
     def _get_bucket(
         self,
         client: Client,
-        bucket_name: str,
-        timeout: TIMEOUT_TYPE,
+        storage_bucket_name: str,
```
Comment (ChameleonTartu) [inline storage_bucket/get.py:39]: @thomasborgen Shouldn't `**kwargs` be passed here as well and down to the `get_bucket`?

File: storage_bucket/delete_file.py [hunk 2]
```diff
@@ -46,10 +46,10 @@ def __call__(
         timeout: Optional[TIMEOUT_TYPE] = None,
```
Comment (ChameleonTartu) [inline storage_bucket/delete_file.py]: Shouldn't it be `**kwargs`?

Comment (ChameleonTartu) [reply storage_bucket/delete_file.py]: I guess that `TIMEOUT_TYPE ` can be removed from the codebase, now. What do you think?

**Review by ChameleonTartu — CHANGES_REQUESTED** [review]
@thomasborgen There are a couple of minor comments.

**Pull request merged**